db/fixtures/miq_user_roles.yml

```diff
@@ -1252,3 +1252,8 @@
   - sui_services_view
   - sui_orders_view
   - sui_notifications
+
+- :name: EvmRole-physical_infrastructure
+  :read_only: true
+  :miq_product_feature_identifiers:
+  - everything
```

Comment:
Are you sure we need to give users in a group having this role "everything"? That is the equivalent of a super administrator. Is there a subset of features that these users would need instead of everything?

Since the purpose of this is mainly to give the group a custom dashboard, perhaps it would be good enough to copy the `EvmRole-administrator` role for this group. Or, maybe you don't need a new role, just a new group that has the new dashboard. What do you think?

Reply:
@gtanzillo We wanted a usergroup whose members have the same access to everything in the UI that the _admin_ user has but with a default dashboard containing our widgets. Is there a better way to do this?

Reply:
@skovic I think this is fine if you need the group to have `everything`. I was thinking that you could just use an existing role with the new group. But, I noticed that the role maps are one to one in naming between group and role, by convention. We might as well keep it that way.